pygmt/src/grdclip.py

```diff
@@ -0,0 +1,89 @@
+"""
+grdclip - Clip a grid.
+"""
+
+import xarray as xr
+from pygmt.clib import Session
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    G="outgrid",
+    R="region",
+    Sa="above",
+    Sb="below",
+    Si="interval",
+    Sr="old",
+    V="verbose",
+)
+@kwargs_to_strings(
+    R="sequence",
+    Sa="sequence",
+    Sb="sequence",
+    Si="sequence",
+    Sr="sequence",
+)
+def grdclip(grid, **kwargs):
+    r"""
+    Sets specific values, or values above/below a set number, to a given value.
+
+    Produce a new ``outgrid`` file clipped version of``grid``.
```

Comment:
Just another suggestion, feel free to improve the wording a bit before committing.
```suggestion
    Produce a new ``outgrid`` file or xarray.DataArray grid which is a clipped
    version of the input ``grid``.
```